hrui/hrui.php

```diff
@@ -594,3 +615,51 @@ function hrui_civicrm_alterContent( &$content, $context, $tplName, &$object ) {
    </script>";
   }
 }
+
+/**
+ * @param int $contractID contract ID
+ * @return string|null return an comma separated string containing the list of departments
```

Comment:
Hello @omar-compucorp 
Maybe we should put description on top of the function instead of putting it with @return.
so we could change it to:
```
/**
* Returns a comma separated string containing the list of departments
*
* @param int $contractID contract ID
* @return string|null
*/
```
Please see other functions for that as well.
Thanks!